core/client-internal/src/main/java/tachyon/client/block/BlockStoreContext.java

```diff
@@ -270,6 +272,7 @@ public boolean hasLocalWorker() {
    * Re-initializes the {@link BlockStoreContext}. This method should only be used in
    * {@link ClientContext}.
    */
+  @SuppressFBWarnings
```

Comment:
This method is supposed to be called from test. If we annotate synchronized, then other fields like `mBlockMasterClientPool` will be checked by findbugs, and it will requires basically all the methods in this class to be synchronized.